src/main/java/io/openliberty/tools/intellij/LibertyExplorer.java

```diff
@@ -220,6 +222,33 @@ public static Tree buildTree(Project project, Color backgroundColor) {
             }
         });
 
+        tree.addMouseMotionListener(new MouseMotionAdapter() {
+            private String currentTooltipText = null;
+
+            @Override
+            public void mouseMoved(MouseEvent e) {
+                super.mouseMoved(e);
+
+                TreePath path = tree.getPathForLocation(e.getX(), e.getY());
+                if (path == null ) {
+                    if (currentTooltipText != null) {
+                        tree.setToolTipText(null);
+                        currentTooltipText = null;
+                    }
+                } else {
+                    Object node = path.getLastPathComponent();
+                    if (node instanceof LibertyModuleNode) {
+                        LibertyModuleNode treeNode = (LibertyModuleNode) node;
+                        String tooltipText = getBuildPath(treeNode);
```

Comment:
Instead of using a new method here, it is best to use available ones. It looks like your new `getBuildPath()` method was written to get the build file, convert it to a String and then replace part of the String to get the correct file path. 

It would be simpler to call the LibertyModuleNode method `getFilePath()` from here, which retrieves a VirtualFile object (this method is confusingly named since it does NOT return a file path, but a file object), and then with the VirtualFile object, you can call `getPath()` to retrieve the file path in a String format to then display to the user.

For example:

```
VirtualFile buildFile = libertyNode.getFilePath();
String filePath = buildFile.getPath();
String tooltipText = filePath;
```
or more condensed:
```
String tooltipText = libertyNode.getFilePath().getPath();
```

Reply:
done.